pkg/machine/e2e/init_test.go

```diff
@@ -97,13 +98,16 @@ var _ = Describe("podman machine init", func() {
 		Expect(inspectBefore).ToNot(BeEmpty())
 		testMachine := inspectBefore[0]
 		Expect(testMachine.Name).To(Equal(mb.names[0]))
-		Expect(testMachine.Resources.CPUs).To(Equal(uint64(cpus)))
-		Expect(testMachine.Resources.Memory).To(Equal(uint64(2048)))
+		if testProvider.VMType() != machine.WSLVirt { // memory and cpus something we can set with WSL
+			Expect(testMachine.Resources.CPUs).To(Equal(uint64(cpus)))
+			Expect(testMachine.Resources.Memory).To(Equal(uint64(2048)))
+		}
 		Expect(testMachine.SSHConfig.RemoteUsername).To((Equal(remoteUsername)))
 
 	})
 
 	It("machine init with cpus, disk size, memory, timezone", func() {
+		skipIfWSL("setting hardware resource numbers and timezone are not supported on WSL")
```

Comment:
How does it behave on WSL?  Does it error out?  Any chance we can test against the expected behavior?

Reply:
i have created #20175 for this, ok?

Reply:
Thanks!